index.html

```diff
@@ -2700,25 +2667,25 @@ <h3><dfn>Writing or pushing content</dfn></h3>
             |message|'s records, run the following steps, or make sure
             that the underlying platform provides equivalent values to
             |ndef|:
-            <ol>
+            <ol data-link-for="NDEFRecordInit">
               <li>
-                If |record|'s recordType is `undefined`:
+                If |record|'s <a>recordType</a> is `undefined`:
```

Comment:
Why not just use [=recordType=]

Reply:
`[=` is for concept links IIUC.

Reply:
No, it is just modern Bikeshed syntax :)

Reply:
ReSpec [says](https://github.com/w3c/respec/wiki/ReSpec-Editor's-Guide) it's for concept links, so I thought normal linking is more appropriate.
In a separate PR, you could replace with `[= =`] if you want :).